api/repositories/service_binding_repository_test.go

```diff
@@ -49,12 +53,30 @@ var _ = Describe("ServiceBindingRepo", func() {
 			korifiv1alpha1.CFServiceBindingList,
 			*korifiv1alpha1.CFServiceBindingList,
 		]{}
+		brokerClient = new(fake.BrokerClient)
+		brokerClient.GetServiceBindingReturns(osbapi.BindingResponse{
```

Comment:
Mocking the brokerClient to return binding parameters should be only needed in the `GetServiceBindingParameters` describe, so this line would be a better fit in the `BeforeEach` there